src/components/Panel/Panel.tsx

```diff
@@ -51,539 +12,29 @@ const errorBackgroundColor = addOpacityToHsla(red500, 0.05);
-    !alreadyProposed &&
-    !alreadySettled;
-  const showConnectButton =
-    page !== "settled" &&
-    primaryAction?.title === connectWallet &&
-    !alreadyProposed &&
-    !alreadySettled;
-
-  useEffect(() => {
-    setMessage("");
-    if (!content || page === "settled") return;
-
-    if (!alreadyProposed && !alreadySettled) return;
-
-    const message = (
-      <>
-        This query has already been {alreadyProposed ? "proposed" : "settled"}.
-        View it{" "}
-        <MessageLink
-          href={{
-            pathname: alreadyProposed ? "/" : "/settled",
-            query: makeUrlParamsForQuery(content),
-          }}
-        >
-          here.
-        </MessageLink>
-      </>
-    );
-
-    setMessage(message);
-  }, [page, content, alreadyProposed, alreadySettled]);
-
-  function getActionsTitle() {
-    if (page === "settled") return "Settled as";
-    if (oracleType === "Optimistic Oracle V3")
-      return (
-        <>
-          Assertion <span>(proposal)</span>
-        </>
-      );
-    return (
-      <>
-        Request <span>(price)</span>
-      </>
-    );
-  }
 
   function close() {
     void closePanel();
   }
 
-  // todo: @sean update copy
-  const bondInformation = (
-    <>
-      <p>
-        Every request to UMA&apos;s Optimistic Oracle includes bond settings
-        that specify the size of the bond that proposers (and disputers) are
-        required to post.
-      </p>
-      <br />
-      <p>The minimum bond is the final fee for a given bond token.</p>
-      <br />
-      <MessageLink
-        href="https://docs.uma.xyz/developers/setting-custom-bond-and-liveness-parameters"
-        target="_blank"
-      >
-        Learn more
-      </MessageLink>
-    </>
-  );
-
-  // todo: @sean update copy
-  const rewardInformation = (
-    <>
-      <p>
-        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Illum, beatae.
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod.
-      </p>
-      <br />
-      <p>
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Blanditiis,
-        mollitia!
-      </p>
-      <br />
-      <MessageLink
-        href="https://docs.uma.xyz/developers/setting-custom-bond-and-liveness-parameters"
-        target="_blank"
-      >
-        Learn more
-      </MessageLink>
-    </>
-  );
-
-  // todo: @sean update copy
-  const livenessInformation = (
-    <>
-      <p>
-        Every request to UMA&apos;s Optimistic Oracle includes liveness settings
-        that specify the liveness window, which is the challenge period during
-        which a proposal can be challenged.
-      </p>
-      <br />
-      <p>A typical liveness window is two hours.</p>
-      <br />
-      <MessageLink
-        href="https://docs.uma.xyz/developers/setting-custom-bond-and-liveness-parameters"
-        target="_blank"
-      >
-        Learn more
-      </MessageLink>
-    </>
-  );
+  const props = !!content
```

Comment:
now the panel's only job is to wrap the other stuff in a panel and render it. setting the props like I've done here lets us do the not undefined assertion in one place, and then treat the query as defined in all the children. simplifies things quite a bit while keeping the logic the same.